src/extension.ts

```diff
@@ -216,8 +217,17 @@ export async function activate(context: vscode.ExtensionContext) {
         await loginWithMostRecentCredentials(toolkitSettings, loginManager)
 
         recordToolkitInitialization(activationStartedOn, getLogger())
+
+        console.log(`xxx activate END PID=${process.pid} getLogger()${getLogger()}`)
```

Comment:
"xxx activate?"

and where does a console.log even get put? getLogger throws if it's null so why can we not use the real logger?

Reply:
these log messages will be removed. console.log sinks to stdout, at least when running from CLI.